### ISSUE TITLE
Consider mentioning request header field protections?

### ISSUE THREAD

**Issue by LPardue** [httpwg/http-extensions#2286]
The security considerations pay some good attention to the concerns of the actors that might deal with the headers defined in client-certs. It crossed my mind that you could mention header protection mechanisms, like Signatures, as one way to harden things.

**Comment by bc-pi** [comment]
Yeah, Signatures could be mentioned in there as a possible additional measure.
